src/uwtools/tests/drivers/test_fv3.py

```diff
@@ -35,7 +28,7 @@ def config(tmp_path):
                 "batchargs": {
                     "walltime": "00:02:00",
                 },
-                "executable": "fv3",
+                "executable": str(tmp_path / "fv3"),
```

Comment:
Since I wrote that comment we removed that requirement from those two task functions, but all the tests still pass so I'm leaving this in place in case we need it later.